<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Master</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Video File: </string>
        </property>
        <property name="buddy">
         <cstring>videoFileNameLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="videoFileNameLineEdit">
        <property name="placeholderText">
         <string>Enter a file path ending with .mp4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openVideoButton">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeVideoButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Save Path:</string>
        </property>
        <property name="buddy">
         <cstring>savePathLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="savePathLineEdit">
        <property name="placeholderText">
         <string>Enter a path without a trailing '/'</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="savePathSetButton">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Scale:</string>
        </property>
        <property name="buddy">
         <cstring>scaleSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="scaleSpinBox">
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Interp:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bilinearRadioBox">
        <property name="text">
         <string>Bilinear</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bicubicRadioBox">
        <property name="text">
         <string>Lanczos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="heqCheckBox">
        <property name="text">
         <string>Hist. Eq.</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Gamma:</string>
        </property>
        <property name="buddy">
         <cstring>gammaDoubleSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="gammaDoubleSpinBox">
        <property name="maximum">
         <double>3.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Rotate:</string>
        </property>
        <property name="buddy">
         <cstring>rotateSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="rotateSpinBox">
        <property name="maximum">
         <number>270</number>
        </property>
        <property name="singleStep">
         <number>90</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Crop</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Top:</string>
           </property>
           <property name="buddy">
            <cstring>cropTopSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="cropTopSpinBox">
           <property name="maximum">
            <number>99</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Bottom:</string>
           </property>
           <property name="buddy">
            <cstring>cropBottomSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="cropBottomSpinBox"/>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Left:</string>
           </property>
           <property name="buddy">
            <cstring>cropLeftSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="cropLeftSpinBox"/>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Right:</string>
           </property>
           <property name="buddy">
            <cstring>cropRightSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="cropRightSpinBox"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Free Mode</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Jump To:</string>
           </property>
           <property name="buddy">
            <cstring>jumpToSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="jumpToSpinBox"/>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Step:</string>
           </property>
           <property name="buddy">
            <cstring>stepSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="stepSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Boomerang &amp;&amp; Loop</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>From:</string>
           </property>
           <property name="buddy">
            <cstring>boomLoopFromSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="loopCheckBox">
           <property name="text">
            <string>Loop</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="boomLoopEnableCheckBox">
           <property name="text">
            <string>Enable</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QSpinBox" name="boomLoopFromSpinBox"/>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="boomLoopDelaySpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>25</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Delay:</string>
           </property>
           <property name="buddy">
            <cstring>boomLoopDelaySpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>To:</string>
           </property>
           <property name="buddy">
            <cstring>boomLoopToSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QSpinBox" name="boomLoopToSpinBox"/>
         </item>
         <item row="0" column="3">
          <widget class="QCheckBox" name="reverseCheckBox">
           <property name="text">
            <string>Reverse</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Current Frame Index:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cfiLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveFrameButton">
        <property name="text">
         <string>Save Current Frame</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="locationHorizontalSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>videoFileNameLineEdit</tabstop>
  <tabstop>openVideoButton</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>closeVideoButton</tabstop>
  <tabstop>savePathLineEdit</tabstop>
  <tabstop>savePathSetButton</tabstop>
  <tabstop>scaleSpinBox</tabstop>
  <tabstop>bilinearRadioBox</tabstop>
  <tabstop>bicubicRadioBox</tabstop>
  <tabstop>heqCheckBox</tabstop>
  <tabstop>gammaDoubleSpinBox</tabstop>
  <tabstop>rotateSpinBox</tabstop>
  <tabstop>cropTopSpinBox</tabstop>
  <tabstop>cropBottomSpinBox</tabstop>
  <tabstop>cropLeftSpinBox</tabstop>
  <tabstop>cropRightSpinBox</tabstop>
  <tabstop>jumpToSpinBox</tabstop>
  <tabstop>stepSpinBox</tabstop>
  <tabstop>boomLoopEnableCheckBox</tabstop>
  <tabstop>loopCheckBox</tabstop>
  <tabstop>reverseCheckBox</tabstop>
  <tabstop>boomLoopFromSpinBox</tabstop>
  <tabstop>boomLoopDelaySpinBox</tabstop>
  <tabstop>boomLoopToSpinBox</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>saveFrameButton</tabstop>
  <tabstop>locationHorizontalSlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
